### PR TITLE
Check empty methods and asciidoc fix

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: src/test/java/net/openhft/chronicle/core/JvmTest.java [hunk 1]
```diff
@@ -271,6 +272,7 @@ static class ClassD extends ClassC {
     }
 
     interface InterfaceWithDefaultMethod {
+        @SuppressWarnings("EmptyMethod")
```
Comment (JerryShea) [inline src/test/java/net/openhft/chronicle/core/JvmTest.java:275]: should this be on helloDefault?

File: src/test/java/net/openhft/chronicle/core/JvmTest.java [hunk 2]
```diff
@@ -460,9 +462,11 @@ static class DTO {
     @AnnoAlias
     interface Foo {
 
+        @SuppressWarnings("EmptyMethod")
```
Comment (JerryShea) [inline src/test/java/net/openhft/chronicle/core/JvmTest.java]: required on an interface?

**Review by tgd — CHANGES_REQUESTED** [review]
Just need to check the suppress warnings empty method on interfaces to see if required

File: src/test/java/net/openhft/chronicle/core/JvmTest.java (hunk 1 above)
Comment (peter-lawrey) [reply src/test/java/net/openhft/chronicle/core/JvmTest.java:275]: The warning is that all implementations are empty.

**Review by tgd — APPROVED** [review]

**Pull request merged**